http-transport-binding.md

```diff
@@ -0,0 +1,338 @@
+# HTTP Transport Binding for Cloud Events
+
+Editor: Clemens Vasters (clemensv@microsoft.com)
```

Comment:
I'd prefer to remove this section. Or add the same (c) for all of our docs, but IMO its unnecessary. 

We should, at some point, discuss the versioning scheme we're going to use for these secondary documents. I'm leaning towards having all docs be at the same level, otherwise mixing-n-matching will be kind of awkward.

Reply:
Will do that